ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx

```diff
@@ -111,8 +147,19 @@ function HeroWithImageTemplate(props: HeroWithImageTemplateProps) {
           'row',
           'col-12',
           fullWidth ? 'mx-0' : 'col-sm-10 mx-auto',
-          isLeftImage ? 'flex-row' : 'flex-row-reverse'
+          isLeftImage ? 'flex-row' : 'flex-row-reverse',
+          imageAsBackground ? 'responsive-bg-image' : ''
         )}
+        style={imageAsBackground ? {
```

Comment:
this code section feels like too many lines for a ternary, but I guess it's nice getting to define most of the style props here

Reply:
I don't feel strongly, but I agree it's nice to see it inline. Open to alternatives.